src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -178,12 +193,32 @@ public void onTraversalStart() {
         // TODO if you change here, also change in CreateArrayIngestFiles
         // Get sample name
         final VCFHeader inputVCFHeader = getHeaderForVariants();
+        if (enableVCFHeaderProcessing) {
+            Set<String> nonCommandLineHeaders = new HashSet<>();
+            for (VCFHeaderLine line :  inputVCFHeader.getMetaDataInInputOrder()) {
+                if (line.getKey().contains("CommandLine")) {
+                    Map<String, String> commandLine = new HashMap<>();
+                    commandLine.put(line.getKey(), line.getValue());
+                    allLineHeaders.put(commandLine.toString(), true);
+                }
+                else {
+                    String lineAsString = line.toString();
+                    nonCommandLineHeaders.add(line.toString());
+                }
+            }
+            String allNonCommandLines = StringUtils.join(nonCommandLineHeaders);
```

Comment:
If no delimiter is specified to `StringUtils.join` there will be [no delimiter](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#join-T...-) which seems like perhaps not ideal for readability

Reply:
When I ran it, it added ", " as the delimiter.

Reply:
Surprising but that does appear to be what's in the database...